config/config/src/Config.ts

```diff
@@ -214,6 +214,7 @@ export interface Config {
   strictStorePkgContentCheck: boolean
   managePackageManagerVersions: boolean
   strictDepBuilds: boolean
+  updateInjectedPackagesAfterRun?: boolean | string[]
```

Comment:
This might be better name
```suggestion
  syncInjectedDepsAfterScripts?: boolean | string[]
```

"injected-deps" is what we use in existing settings, not injected packages.

Reply:
or alternatively we could have two options. One for listing "build" scripts and another one to enable injected deps syncing.
```suggestion
  syncInjectedDepsOnBuild?: boolean
  buildScripts?: string[]
```

Reply:
https://github.com/pnpm/pnpm/pull/9100/commits/cb08cf91fb94991ed4afd06741099ae45b7036d3